src/python/requirements.txt

```diff
@@ -11,3 +11,4 @@ rpi-ws281x==4.3.4
 RPi.GPIO==0.7.1
```

Comment:
I think we want to remove this callout as we discussed, since gpiozero has this as a dependency already.